From 2203e9045d8d113e1ecceddcbebcf78c66af557f Mon Sep 17 00:00:00 2001
From: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
Date: Wed, 13 Jul 2016 15:49:12 +0530
Subject: [PATCH 77/93] board: ls1012aqds: Avoid reset masking

FPGA regiser RST_MASK1 control reset masking for I2C, USB,
Mem and EPHY1.

This register was being masked during initial debugging of
RGMII. It is not required

So avoid masking of these device to send reset signals during
soft-reset.

Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
---
 board/freescale/ls1012aqds/eth.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/board/freescale/ls1012aqds/eth.c b/board/freescale/ls1012aqds/eth.c
index 6fbbdbe..04d621c 100644
--- a/board/freescale/ls1012aqds/eth.c
+++ b/board/freescale/ls1012aqds/eth.c
@@ -156,12 +156,6 @@ int board_eth_init(bd_t *bis)
 	data8 |= 0x2;
 	QIXIS_WRITE(rst_frc[0], data8);
 	data8 = QIXIS_READ(rst_frc[0]);
-
-	data8 = QIXIS_READ(res8[6]);
-	data8 |= 0xff;
-	QIXIS_WRITE(res8[6], data8);
-	data8 = QIXIS_READ(res8[6]);
-
 #endif
 
 	mac1_mdio_info.reg_base = (void *)0x04200000; /*EMAC1_BASE_ADDR*/
-- 
1.7.9.5

